dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -503,7 +503,7 @@ class ScriptLevelsControllerTest < ActionController::TestCase
 
   test "should 404 for invalid script level for twenty hour" do
     assert_raises(ActiveRecord::RecordNotFound) do # renders a 404 in prod
-      get :show, script_id: Script::TWENTY_HOUR_ID, id: 40000
+      get :show, script_id: Script::TWENTY_HOUR_NAME, id: 40000
```

Comment:
(this is the old route that you're eliminating/redirecting in the other commit though, since now the canonical route is script/stage/puzzle)